agents/agents/agentsintegration/agentsintegration_test.go

```diff
@@ -5,9 +5,9 @@ import (
 	"github.com/brianvoe/gofakeit/v6"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/synapsecns/sanguine/agents/agents/executor"
-	executorCfg "github.com/synapsecns/sanguine/agents/agents/executor/config"
 	"github.com/synapsecns/sanguine/agents/agents/guard"
 	"github.com/synapsecns/sanguine/agents/agents/notary"
+	executor2 "github.com/synapsecns/sanguine/agents/config/executor"
```

Comment:
can we get rid of these "2" suffixes, if it conflicts w/ a var var should prob be renamed. If it's another package (the case here, just do executorConfig for clarity)

Reply:
done